projects/github-actions/repo-gardening/src/tasks/triage-issues/index.js

```diff
@@ -57,30 +56,28 @@ ${ body }
 
 **********************
 
-And finally, here is a list of the existing labels and their descriptions:
+You must analyze this content, and suggest labels related to the content.
+The labels you will suggest must all come from the list below:
 
 ${ repoLabels
 	.map( label => `- ${ label.name }${ label?.description ? `: ${ label.description }` : '' }` )
 	.join( '\n' ) }
 
 Analyze the issue and suggest relevant labels. Rules:
-- Don't search for and add more labels if the existing labels match your initial assessment.
 - Use only existing labels provided.
 - Include 1 '[Feature Group]' label.
 - Include 1 to 3 '[Feature]' labels.
 - Briefly explain each label choice in 1 sentence.
-- Format your response as a JSON object with 'labels' and 'explanations' keys.
+- Format your response as a JSON object, with each suggested label as a key, and its explanation as the value.
```

Comment:
Would it help to be more explicit here?
```suggestion
- Format your response as a JSON object, with each suggested label as a key, and your explanation of the label choice as the value.
```